shiny-tic-toc.js

```diff
@@ -223,11 +263,10 @@ function plotMeasurements() {
       }
     );
   }
-
   option = {
```

Comment:
Could be `const option = {` here (and delete `var option;` above)`.

Reply:
Addressed in 650fe6a772d4be164082bb84961d96ea9ec69503